core/domain/html_validation_service_test.py

```diff
@@ -917,7 +917,7 @@ def test_validate_customization_args(self):
                 'amp;lt;/p&amp;gt;&amp;quot;, &amp;quot;title&amp;quot;: '
                 '&amp;quot;Savjet 1&amp;quot;}]"></oppia-noninteractive-tabs>'
             )],
-            "Missing attributes: [u'alt-with-value'], Extra attributes: []": [(
+            'Missing attributes: [u\'alt-with-value\'], Extra attributes: []': [( # pylint: disable=line-too-long
```

Comment:
Please do not use pylint pragmas. Instead try breaking this up into multiple lines.